### PR TITLE
Some suggestions for this SSD topic

### PR DESCRIPTION
**Pull request by timothymcmackin**
Here are some suggestions based on my experience with SSDs.

Is there something that needs to be done with the tables in this topic? I ran it through sphinx and it looks OK. Not sure what this note means at the top:
> This one requires effort to apply Restructured Text tags, becuase it has tables

### CODE REVIEW

File: StaticSiteGenerators(tables).rst [hunk 1]
```diff
@@ -14,42 +14,40 @@ Starting notes:
 * What are static site generators?
 * From the June WTD Newletter:
 
-WHAT IS A STATIC SITE GENERATOR
-===============================
+WHAT IS A STATIC SITE GENERATOR?
+================================
 
-A static site generator is a tool that can publish a collection of markup files to a site. Popular static site generators include Sphinx, Jekyll, and Asciidoctor.
+A static site generator (SSD) is a tool that can convert a collection of markup files into web pages. Most SSDs include programs that host the resulting web site, but because the output doesn't depend on a specific hosting server, you have many options for deploying the content. Popular static site generators include Sphinx, Jekyll, and Asciidoctor.
```
Comment (clairealyssa) [inline StaticSiteGenerators(tables).rst]: Why are you using SSD instead of SSG here?

File: StaticSiteGenerators(tables).rst [hunk 2]
```diff
@@ -14,42 +14,40 @@ Starting notes:
...
-* Permissions - Access control isn’t built into most static site generators. If you don’t want your docs publicly accessible, access control is an additional expense on your time. 
-* Versioning - If you’re using a certain kind of markup, you can use readthedocs has versioning built into the theme. Different branches represent different versions. If you name the branch a certain way (for example 3.5). Whenever you’re ready to release a different version of your product, you create a different branch, it creates the version for you.
-* No one has developed this function for Markdown yet. Or it’s not popular yet. 
-* How do you go from version to version and how easy is it for your end users to find different versions of your docs. 
+* SSDs have no content management system to manage things like:
+  * Permissions - Access control isn’t built into most static site generators. If you don’t want your docs publicly accessible, access control is an additional expense on your time.
+  * Versioning - Not all SSDs support deploying different versions of the same content at once. However, some SSDs can use version control to handle deploying different versions simultaneously.
```
Comment (clairealyssa) [inline StaticSiteGenerators(tables).rst]: There's more information in the versioning list item in the original text that I'm not sure bears removing. What are your thoughts on adding another list item or additional explanation outside of the list?

File: StaticSiteGenerators(tables).rst (hunk 1 above)
Comment (timothymcmackin) [reply StaticSiteGenerators(tables).rst]: I... don't know. What a strange brain fart. I'll fix that.

File: StaticSiteGenerators(tables).rst (hunk 2 above)
Comment (timothymcmackin) [reply StaticSiteGenerators(tables).rst]: Ok, I've added back some of that info on versioning. Not sure where else it should go other than that same bullet.

**Pull request merged**